RELEASE.md

```diff
@@ -0,0 +1,127 @@
+# Release procudure for a new version of CF
+
+
+## Update the revision history in file **history.adoc**
+
+  - Check that the history is up to date. Every non-trivial issue or merged pull request without an issue should be referenced.
+  - Replace the line _`=== Working version (most recent first)`_ with the line `=== Version <VN> (<DATE>)`, substituting `<VN>` with the release version and `<DATE>` with the current date (e.g. `Version 1.12 (04 December 2024)`).
+
+
+## Create the new documents
+
+Go to your fork of https://github.com/cf-convention/cf-conventions and create the new documents:
+```
+cd ~/cf-conventions
+export CF_FINAL=True
+make clean
+make all
+```
+
+
+## Copy the new documents to the website repository
+
+Go to your fork of https://github.com/cf-convention/cf-convention.github.io, create a new branch from "main", copy the new documents into this branch and submit the changes as a puull request back to https://github.com/cf-convention/cf-convention.github.io. The following is an example of how this may be done, but other git formulations are possible:
```

Comment:
```suggestion
Go to your fork of https://github.com/cf-convention/cf-convention.github.io, create a new branch from "main", copy the new documents into this branch and submit the changes as a pull request back to https://github.com/cf-convention/cf-convention.github.io. The following is an example of how this may be done, but other git formulations are possible:
```